manticore/core/executor.py

```diff
@@ -491,6 +491,15 @@ def generate_testcase(self, state, message = 'Testcase generated'):
         
         file(self._getFilename('test_%08x.syscalls'%test_number),'a').write(repr(state.platform.syscall_trace))
 
+        # save symbolic files
+        if hasattr(state.platform, 'files'):
+            for f in state.platform.files:
+                if hasattr(f,'array'):
+                    buf = solver.get_value(state.constraints, f.array)
+                    filename = os.path.basename(f.array.name)
+                    filename = 'test_file_%s_%d.txt'%(filename, test_number)
```

Comment:
Should I also directly change all the remaining `%` to `.format` used in `generate_testcase` ?